katello-installer/katello-installer-base.spec

```diff
@@ -9,85 +9,72 @@ Summary: Puppet-based installer for the Katello and Katello Capsule
 Group:   Applications/System
 License: GPLv3+ and ASL 2.0
 URL:     http://katello.org
-Source0: https://github.com/Katello/katello-installer/archive/%{version}.tar.gz
+Source0: https://github.com/Katello/katello-installer/archive/katello-installer-%{version}.tar.gz
 
 BuildArch: noarch
-Conflicts: foreman-installer
 Obsoletes: katello-installer < 2.1.0
```

Comment:
Could we? `foreman-installer-capsule` oboletes old `katello-installer` only via this line.
This could cover situation when someone installing new capsule scenario directly over `katello-installer < 2.1.0`. Not sure if this is supported.